### ISSUE TITLE
Log autoscroll (toggleable)

### ISSUE THREAD

**Issue by itssimple** [dotnet/tye#672]
### What should we add or change to make your life better?

It would be nice if you could toggle if the log window should tail (scroll down when new logs appears), so you can keep the window open and see all new log items as they come in without having to scroll down manually.

### Why is this important to you?

This would make it easier for devs to toggle if they want to read the log at their own pace, or if they think it's more important with seeing the log as it comes in without scrolling.

**Comment by tebeco** [comment]
Yeah a toggle on this is mandatory, because forced scroll is probably worse than the current "no scroll" :D

**Comment by jammehcow** [comment]
Should be addressed by #1001 which was merged a year and a bit ago
